### PR TITLE
V0.2.0 M4

### PR DESCRIPTION
**Pull request by FabrizioMoggio**
M4 version for Fall24:

- OAS file renamed according to: https://github.com/camaraproject/CallForwardingSignal/issues/93
- Remove “-rc.n” from the version and server path: https://github.com/camaraproject/CallForwardingSignal/discussions/92
- Remove "-rc.02" from "x-camara-commonalities": https://github.com/camaraproject/CallForwardingSignal/issues/94
- updated changelog.md
- updated readme.md
- updated .feature files according to: https://github.com/camaraproject/CallForwardingSignal/issues/96
- updated Readiness CL according to: https://github.com/camaraproject/CallForwardingSignal/issues/97
- updated changelog.md according to: https://github.com/camaraproject/CallForwardingSignal/issues/98
- updated readme.md according to: https://github.com/camaraproject/CallForwardingSignal/issues/100

#### What type of PR is this?

Add one of the following kinds:
* cleanup
* documentation

#### What this PR does / why we need it:
final release for Fall24

#### Which issue(s) this PR fixes:
https://github.com/camaraproject/CallForwardingSignal/issues/94
https://github.com/camaraproject/CallForwardingSignal/issues/93
https://github.com/camaraproject/CallForwardingSignal/issues/96
https://github.com/camaraproject/CallForwardingSignal/issues/97
https://github.com/camaraproject/CallForwardingSignal/issues/98
https://github.com/camaraproject/CallForwardingSignal/issues/100

### CODE REVIEW

**Comment by hdamker** [comment]
@FabrizioMoggio thanks for the release PR - have created https://github.com/camaraproject/ReleaseManagement/issues/77 for the Release Management review. 

Please update also the API Readiness checklist and the version within the .feature files as part of the PR.

**Comment by hdamker** [comment]
@FabrizioMoggio and another important point: 

The Release Notes for a public release must be in generell a summary of all changes since the last public release, not just the delta to the last release candidate.
In your case - as there was no previous public release and everything is new - it's more a list of "What's new" as you had in the first release candidate - you don't need to mention the corrections of this new stuff which happened between the release candidates, as for someone who is taking this public release, everything is anyway new.

Note: the "full changelog" link is
`Full Changelog: https://github.com/camaraproject/CallForwardingSignal/commits/r1.3`

**Review by bigludo7 — DISMISSED** [review]
LGTM
Your PR will be used as an example for me for other API Fabrizio :)

**Comment by bigludo7** [comment]
Correction for my comment: this @FabrizioMoggio PR will be the example once the comment from @hdamker will be fixed ;)

**Review by hdamker — CHANGES_REQUESTED** [review]
* Please update also the API Readiness checklist and the version within the .feature files as part of the PR
* Complete the CHANGELOG for the public release with all updates, see my previous comment

**Comment by hdamker** [comment]
BTW: release tracker updated ... r1.3 will be the public release tag, not the next pre-release.

**Comment by FabrizioMoggio** [comment]
I’m sorry to ask for advice, I’m trying do to my best 😊 from my holidays. For M4 I should have addressed all your comments. The current PR for the CFS API is targeting the public release. What I’m not sure of is if at 30/08 the public release should be merged (if so, we missed it 😊) or if the final approval from @camaraproject/release-management_maintainers is due for M5 (09-15).

**Comment by hdamker** [comment]
@FabrizioMoggio thank for taking car during your vacation! No worries, you have done every necessary for the M4 milestone by creating and updating this release PR so far. Indeed, the review and approval by Release Management will happen before M5 and then the merge.

@bigludo7 @chinaunicomyangfan Please review the PR and the latest changes as well and approve the PR already.

**Review by bigludo7 — DISMISSED** [review]
LGTM

**Review by FabrizioMoggio — COMMENTED** [review]
the requested changes by @hdamker have been implemented

**Comment by FabrizioMoggio** [comment]
> * Please update also the API Readiness checklist and the version within the .feature files as part of the PR
> * Complete the CHANGELOG for the public release with all updates, see my previous comment

Dear @hdamker I don't know how to move forward. You requested a change and I have implemented it but I can't find a way to mark the requested changes as implemented. So I can't merge. Is it you that need to control and mark them as implemented?

**Review by bigludo7 — APPROVED** [review]
LGTM

File: documentation/API_documentation/call-forwarding-signal-API-Readiness-Checklistst.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 
 # API Readiness Checklist
 
-Checklist for API call-forwarding-signal v0.2.0-rc.2 in r1.2
+Checklist for API call-forwarding-signal v0.2.0 in r1.3
 
 
 | Nr | API release assets  | alpha | release-candidate |  public-release<br>initial | public-release<br> stable | Status | Comments |
 |----|----------------------------------------------|:-----:|:-----------------:|:-------:|:------:|:----:|:----|
 |  1 | API definition                               |   M   |         M         |    M    |    M   |   Y  | /code/API_definitions/Call_Forwarding_Signal.yaml |
```
Comment (tanjadegroot) [inline documentation/API_documentation/call-forwarding-signal-API-Readiness-Checklistst.md:9]: change to new API yaml file name (call-forwarding-signal.yaml)

**Review by tanjadegroot — CHANGES_REQUESTED** [review]
a few broken links and typos to fix and this is OK: LGTM from release mgmt

**Comment by FabrizioMoggio** [comment]
Closed because EasyCLA problems.

Replaced by: https://github.com/camaraproject/CallForwardingSignal/pull/101

**Pull request closed without merging**